scripts/monitoring/cron-send-rpms-versions.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Check OpenShift version
+OC_VERSION=$(oc version | sed -n '1{p;q}' | sed 's/[^0-9.]*\([0-9.]*\).*/\1/')
+
+# Check Docker version
+DOCKER_VERSION=$(docker --version | sed 's/[^0-9.]*\([0-9.]*\).*/\1/')
```

Comment:
The problem with this is that it's running inside of our container.

That means that you're checking the version of docker in the container and not on the host.

We bind in the docker socket, so you should be able to use docker-py to talk to the socket and get the version from there.

Does that sound right?